xbmc/addons/kodi-addon-dev-kit/include/kodi/libXBMC_addon.h

```diff
@@ -478,6 +479,19 @@ namespace ADDON
     }
 
     /*!
+    * @brief Get multiple properties from an open file.
+    * @param file The file to the an properties for
+    * @param type type of the requested properties.
+    * @param name of the requested properties / can be null.
+    * @param numProperties number of properties returned.
```

Comment:
Please make the first letter after the parameter name capital - applies  to above three lines.